src/factories/routePathFactory.ts

```diff
@@ -49,6 +49,9 @@ class RoutePathFactory {
     }
 
     public static createNewRoutePath(lineId: string, route: IRoute): IRoutePath {
+        const defaultDate = new Date();
+        defaultDate.setHours(0, 0, 0, 0);
```

Comment:
Move to constants? Something like:
NEW_ROUTE_PATH_HOURS: 0

Reply:
Standard dates are always written as midnight, so I would assume this will never change, and therefore better be hard coded.